test/integration/cache_management_test.rb

```diff
@@ -15,28 +15,29 @@ class CacheManagementTest < ActionDispatch::IntegrationTest
                                               x: [1, 4, 6],
                                               y: [7, 5, 3],
                                               z: [2, 8, 9],
-                                              cells: ['A', 'B', 'C']
+                                              cells: %w[A B C]
                                             },
                                             x_axis_label: 'PCA 1',
                                             y_axis_label: 'PCA 2',
                                             z_axis_label: 'PCA 3',
                                             cluster_type: '3d',
                                             annotation_input: [
-                                              { name: 'Category', type: 'group', values: ['bar', 'bar', 'baz'] },
+                                              { name: 'Category', type: 'group', values: %w[bar bar baz] },
                                               { name: 'Intensity', type: 'numeric', values: [1.1, 2.2, 3.3] }
                                             ])
 
-    @study_exp_file = FactoryBot.create(:study_file,
+    @study_exp_file = FactoryBot.create(:expression_file,
                                         name: 'dense.txt',
+                                        cell_input: %w[A B C],
                                         file_type: 'Expression Matrix',
                                         study: @study)
 
     @study_metadata_file = FactoryBot.create(:metadata_file,
                                              name: 'metadata.txt', study: @study,
-                                             cell_input: ['A', 'B', 'C'],
+                                             cell_input: %w[A B C],
```

Comment:
Can we change Robocop configuration to do the opposite?  This impedes reading flow for folks much more familiar with JavaScript or Python than Ruby.

Reply:
To be fair... we're writing Ruby here, not Python or Javascript.  Both Python and Javascript have ways of notating things (like list comprehensions, or the `...` notation in ES6) that initially can be difficult to understand, but are considered idiomatic to those languages.  However, once I can grasp the context, their utility becomes obvious.  This is the same - Ruby has a [quite a few literals](https://docs.ruby-lang.org/en/2.6.0/syntax/literals_rdoc.html#label-Percent+Strings) for creating objects, like arrays of strings/symbols, regular expressions, and so on.  The edited syntax has dramatically fewer characters, and is _sooo_ much easier to type, rather than constantly opening/closing quotes.

I guess the TL;DR is "I'd prefer not to, because this is how the Ruby community indicates it to be written".  While not all linter rules make the same amount of sense, I think that this is a very good one.  For context - we've had the same discussion about `unless` vs. negative `if`.  I think idiomatic shortcuts like this help to increase our familiarity with languages we may not be as facile in.

Reply:
That's fair!  I think this comes down to taste; differing prioritization of idiomaticness vs. broad reading ease.